File: Python/beautiful-days-at-the-movies.py
def beautifulDays(i, j, k):
    count = 0
    for i in range(i,j+1):
        num = i
        rev = 0
        while num > 0:
            temp = num % 10
            num = num //10
            rev = rev*10 + temp
        val = abs(rev-i)
        if val % k ==0:
            count+=1
    return count

File: Java/beautiful-days-at-the-movies.java
class Result {
    public static int beautifulDays(int i, int j, int k) {        
        int count = 0;
        for(int x=i;x<=j;x++){
            int num = x;
            int rev = 0;
            while(num>0){
                int temp = num % 10;
                num = num/10;
                rev = (rev* 10) + temp;
            }
            int val = Math.abs(rev-x);
            if(val % k == 0) count++;
        }
        return count;
    }
}